Jenkinsfile

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env groovy
+
+@Library('lco-shared-libs@0.0.1') _
+
+pipeline {
+	agent any
+	environment {
+		dockerImage = null
+		PROJ_NAME = projName("${JOB_NAME}")
+		GIT_DESCRIPTION = gitDescription()
+		DOCKER_IMG = dockerImageName("${LCO_DOCK_REG}", "${PROJ_NAME}", "${GIT_DESCRIPTION}")
+	}
+	stages {
+		stage('Build image') {
+			steps {
+				script {
+					dockerImage = docker.build("${DOCKER_IMG}")
```

Comment:
You can pass build args like so `docker.build("${DOCKER_IMG}", "--build-arg CONDA_VER=4.3.21-6-ga718de9 .")`